WebApiCoreSeed.UnitTests/Services/UserServiceTests.cs

```diff
@@ -0,0 +1,59 @@
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.EF;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.Domain.Services;
+using Xunit;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    public class UserServiceTests
+    {
+        [Fact]
+        public async Task GetByIdAsync_ShouldGetUserById_WhenUserIdExists()
+        {
+            // Arrange
+            var optionsBuilder = new DbContextOptionsBuilder<WebApiCoreSeedContext>();
+            optionsBuilder.UseInMemoryDatabase();
+            var createdUser = new User { Id = new Guid("d58893fd-3836-4308-b840-85f4fe548264"), FirstName = "John", LastName = "Doe", Email = "JohnDoe@makinsense.com", UserName = "JohnDoe" };

```

Comment:
Can you split this object initializer into multiple lines? This particular line is too long.